### PR TITLE
Known issues for nrf53

### PR DESCRIPTION
**Pull request by jostn**

### CODE REVIEW

**Comment by jostn** [comment]
This issue is related to https://github.com/zephyrproject-rtos/zephyr/issues/30441

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -152,6 +152,10 @@ nRF5340
 FOTA does not work
   FOTA with the :ref:`zephyr:smp_svr_sample` does not work.
 
+The hci_uart-example supports maximum 69 bytes of HCI Data packet payload.
+  **Workaround** Modify :file:`ncs/zephyr/include/bluetooth/hci_raw.h`::
+  Change BT_L2CAP_MTU from 65 to 247.
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:157]:
```suggestion
Limited HCI data packet payload in HCI UART sample
  The :ref:`zephyr:bluetooth-hci-uart-sample` sample supports a maximum of 69 bytes of HCI data packet payload.
  
  **Workaround:** In the :file:`ncs/zephyr/include/bluetooth/hci_raw.h` file, change the value for ``BT_L2CAP_MTU`` from 65 to 247.
```

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -152,6 +152,10 @@ nRF5340
 FOTA does not work
   FOTA with the :ref:`zephyr:smp_svr_sample` does not work.
 
+The hci_uart-example supports maximum 69 bytes of HCI Data packet payload.
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:155]: Is there a JIRA issue for this?

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -152,6 +152,10 @@ nRF5340
 FOTA does not work
   FOTA with the :ref:`zephyr:smp_svr_sample` does not work.
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:154]: This entry should be moved to the Zephyr section.

**Review by ru-fu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**